src/styles/stars.scss

```diff
@@ -0,0 +1,22 @@
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  background: url("../images/star.svg");
+  background-repeat: no-repeat;
+  background-position: center;
+
+  width: 16px;
+  height: 16px;
+  margin-bottom: 0;
+  margin-right: 4px;
+}
+
+@for $star from 1 through 4 {
+  .stars--#{$star}{
+    .stars__star:nth-child(-n + #{$star}) {
+      background-image: url(../images/star-active.svg);
+    }
+  }
+}
```

Comment:
Doesn't work for stars--5 modifier
![image](https://github.com/mate-academy/layout_catalog/assets/54936044/774b5c1e-b482-48b0-9fa9-fce3314660c2)